types/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "devlaunchers-strapi",
+  "name": "@devlaunchers/strapi-test",
```

Comment:
@chungthuang do you know which change this is related to? CC: @nikita-pardeshi-github @VictorDiniz89